### ISSUE TITLE
Define a default transcode tmp directory

### ISSUE THREAD

**Issue by ljm42** [limetech/dockerapp-plex#2]
One of the problems users have with the various Plex dockers on unRAID is that they transcode in the docker.img by default.  There is documentation to work around this:
  https://lime-technology.com/forum/index.php?topic=40937.msg419495#msg419495
but it requires each user to manually implement it.  

Would you please consider implementing an automatic fix for this?  Binhex already does it:
  https://github.com/binhex/arch-plex/blob/master/setup/nobody/start.sh
and I've made a similar request for LS.IO. You basically need to create a /config/transcode (or /config/tmp) directory and set some environment variable so that Plex will use it:
    mkdir -p /config/tmp
    export PLEX_MEDIA_SERVER_TMPDIR='/config/tmp'
    export TMPDIR='/config/tmp'

Plex has documentation for this here:
  https://support.plex.tv/hc/en-us/articles/200273978-Linux-User-and-Storage-configuration

Thanks!

**Comment by eschultz** [comment]
Good idea.  In testing, with the latest version of Plex, I wasn't able to override the transcoding temp files location to use `/config/tmp` no matter what I tried :angry: 

Might be a change in the latest Plex but it no longer writes to a /tmp folder inside the docker.img image and instead uses `/config/Library/Application Support/Plex Media Server/Cache/Transcode/Sessions/` ... so I guess this takes care of it.

I've submitted the new Dockerfile to Docker Hub so the new release should be available in under an hour.

**Comment by ljm42** [comment]
Very interesting.  I installed a fresh version of the Limetech PlexMediaServer ( 0.9.16.3 ) into an empty /config directory and started watching a movie via a web browser.  I set the quality to 2mbps to force it to transcode.

I "docker exec'd" into the container and used "ps -aef" to find the pid for "./Plex Media Server" 
and then ran this command to see what files it had open:
  ls -l /proc/25/fd
And what do you know... it was transcoding to this directory:
  /config/Library/Application Support/Plex Media Server/Cache/Transcode/Sessions/.../
There were no open files in /tmp or anything outside of /config, really.

So as of Plex 0.9.16.3 (maybe earlier), it seems we don't need to worry about Plex filling up the docker.img file.

Agreed?

I reviewed the Plex release notes and didn't find a mention of this specifically, but there have been a number of transcoder-related changes lately.

**Comment by eschultz** [comment]
Agreed.  I saw the exact same behavior you did with the transcode path.  I also attempted to find that change in the release notes and documentation but came up short.

To ensure all transcoding temp files are created outside Docker.img, our Plex container is still performing these operations just in-case the undocumented transcode folder change is reverted in a future release:

```
mkdir -p /config/tmp
export PLEX_MEDIA_SERVER_TMPDIR='/config/tmp'
export TMPDIR='/config/tmp'
```
